events-api/src/modules/events/events.service.ts

```diff
@@ -61,8 +53,73 @@ export class EventsService {
       },
       skip: (page - 1) * limit,
       take: limit,
+      orderBy: { [sortBy]: sortOrder },
     });
 
     return events;
   }
+
+  async updateEvent(eventId: string, dto: UpdateEventReqDto) {
+    await this.verifyEventExistence(eventId);
+
+    if (dto.eventName) {
+      await this.verifyNameUniqueness(dto.eventName);
```

Comment:
Again, I'd suggest to return a boolean from this method. And then explicitly throw an error

Reply:
> Again, I'd suggest to return a boolean from this method. And then explicitly throw an error

fixed